lib/ansible/modules/packaging/language/maven_artifact.py

```diff
@@ -60,11 +62,11 @@
         required: true
     artifact_id:
         description:
-            - The maven artifactId coordinate
+            - The maven artifactId coordinate.
```

Comment:
The diff looks good, but this isn't picking up the changes properly. Is this something I can fix?